src/CMakeLists.txt

```diff
@@ -701,31 +702,31 @@ if (${NUPIC_BUILD_PYEXT_MODULES})
   # nupic/nupic.bindings presently have no dependency on pycapnp. We also include
   # PYTHON_LIBRARIES because all symbols have to resolve when building a DLL.
   if("${PLATFORM}" STREQUAL "windows")
-    list(APPEND src_swig_link_libraries
+    list(APPEND _SRC_SWIG_LINK_LIBRARIES
          ${PYTHON_LIBRARIES}
          ${CAPNP_STATIC_LIB_TARGET})
   endif()
 
-  message(STATUS "src_swig_extra_deps     = ${src_swig_extra_deps}")
-  message(STATUS "src_swig_link_libraries = ${src_swig_link_libraries}")
-  message(STATUS "src_swig_extension_link_flags= ${src_swig_extension_link_flags}")
+  message(STATUS "_SRC_SWIG_EXTRA_DEPS     = ${_SRC_SWIG_EXTRA_DEPS}")
+  message(STATUS "_SRC_SWIG_LINK_LIBRARIES = ${_SRC_SWIG_LINK_LIBRARIES}")
+  message(STATUS "_SRC_SWIG_EXTENSION_LINK_FLAGS= ${_SRC_SWIG_EXTENSION_LINK_FLAGS}")
   message(STATUS "CMAKE_SWIG_FLAGS        = ${CMAKE_SWIG_FLAGS}")
 
 
   function(PREPEND_BOILERPLATE_TO_PYTHON_PROXY_MODULE
-           PARENT_SWIG_TARGET)
-    # Add a custom command to the given Swig target to prepend boilerplate to
-    # the swig-generated python proxy module
-    # ${CMAKE_SWIG_OUTDIR}/${PARENT_SWIG_TARGET}.py. The boilerplate preloads
+           MODULE_NAME)
```

Comment:
Renamed after getting a better understanding that the value was not a target, but just the swig module name. The target name is actually `${SWIG_MODULE_${MODULE_NAME}_REAL_NAME}`